MaintenanceManager/MaintenanceManager.cpp

```diff
@@ -310,7 +310,6 @@ namespace WPEFramework {
         void MaintenanceManager::InitializeIARM()
         {
             if (Utils::IARM::init()) {
-                LOGINFO();
                 IARM_Result_t res;
                 IARM_CHECK(IARM_Bus_Connect());
```

Comment:
Pls remove this IARM_CHECK(IARM_Bus_Connect());
This is done within Utils::IARM::init().

Reply:
Removed and updated